package.json

```diff
@@ -36,7 +36,7 @@
     "inquirer": "7.0.0",
     "jmespath": "0.15.0",
     "js-yaml": "^3.13.1",
-    "laundry-cfn": "1.3.0",
+    "laundry-cfn": "v1.4.0",
```

Comment:
```suggestion
    "laundry-cfn": "1.4.0",
```

right?

Reply:
Right. cut and pasting from the tag got me in trouble. Pushing the fix